kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/MobiComKitBroadcastReceiver.java

```diff
@@ -125,6 +125,8 @@ public void onReceive(Context context, Intent intent) {
             conversationUIService.updateMessageMetadata(keyString);
         } else if (BroadcastService.INTENT_ACTIONS.MUTE_USER_CHAT.toString().equals(action)) {
             conversationUIService.muteUserChat(intent.getBooleanExtra("mute", false), intent.getStringExtra("userId"));
+        } else if(BroadcastService.INTENT_ACTIONS.AGENT_STATUS.toString().equals(action)) {
+            conversationUIService.updateAgentStatus(intent.getStringExtra("userId"), intent.getIntExtra("status", 3));
```

Comment:
here 3 is agent status online right?If yes Can you use `STATUS_ONLINE` to avoid confusion in future

Reply:
@Sathyan-Elangovan I have updated it